projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-layout.php

```diff
@@ -1,24 +1,80 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+/**
+ * Tiled gallery layout class.
+ */
 abstract class Jetpack_Tiled_Gallery_Layout {
-	// Template allow list.
+	/**
+	 * Template allow list.
+	 *
+	 * @var array
+	 */
 	private static $templates = array( 'carousel-container', 'circle-layout', 'rectangular-layout', 'square-layout' );
-	private static $partials  = array( 'carousel-image-args', 'item' );
 
-	protected $type; // Defined in child classes
+	/**
+	 * Partial list.
+	 *
+	 * @var array
+	 */
+	private static $partials = array( 'carousel-image-args', 'item' );
+
+	/**
+	 * Type of gallery - defined in parent class.
+	 *
+	 * @var string
+	 */
+	protected $type;
+
+	/**
+	 * The attachments.
+	 *
+	 * @var object
+	 */
 	public $attachments;
+
+	/**
+	 * The attachment link.
+	 *
+	 * @var string
+	 */
 	public $link;
+
+	/**
+	 * If the image is in grayscale.
+	 *
+	 * @var bool
+	 */
 	public $grayscale;
+
+	/**
+	 * How many columns.
+	 *
+	 * @var int
+	 */
 	public $columns;
-	public function __construct( $attachments, $link, $grayscale, $columns ) {
 
+	/**
+	 * Constructor function.
+	 *
+	 * @param object $attachments - the attachmed image.
+	 * @param string $link - the attachment link.
+	 * @param bool   $grayscale - if the image is in grayscale.
+	 * @param int    $columns - how many columns.
+	 */
+	public function __construct( $attachments, $link, $grayscale, $columns ) {
 		$this->attachments           = $attachments;
 		$this->link                  = $link;
-		$this->needs_attachment_link = ! ( isset( $link ) && $link == 'file' );
+		$this->needs_attachment_link = ! ( isset( $link ) && $link === 'file' );
 		$this->grayscale             = $grayscale;
 		$this->columns               = $columns;
 	}
 
-	public function HTML( $context = array() ) {
+	/**
+	 * Render carousel container template.
+	 *
+	 * @param array $context - the context.
+	 */
+	public function HTML( $context = array() ) { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
```

Comment:
Would it be worth renaming the method here and where it's being used, so we can get rid of the phpcs ignore?

Reply:
I was worried because all of the child classes also define `HTML` (but for some reason, the sniff only complains about the parent class here), so I think I'd have to update those too. I was simply afraid I miss renaming everything. If you think it's worth it, I can give it a go!

Reply:
We can do that in a follow-up PR I suppose. 👍